pyaedt/edb_core/layout.py

```diff
@@ -166,12 +162,12 @@ def paths(self):
 
     @property
     def bondwires(self):
-        """
+        """Retrieve all bondwires.
```

Comment:
```suggestion
        """List of bondwires.
```

Reply:
Oops, didn't see you were editing this, sorry!